src/components/Jobs/JobListFilters.vue

```diff
@@ -58,13 +69,11 @@ withDefaults(defineProps<FiltersProps>(), {
       justify-content: center;
 
       &-button {
```

Comment:
if its nested it should be modification. if it's a separate element -> don't make it nested. &__clear-button